osf/models/preprint.py

```diff
@@ -303,15 +308,15 @@ def create(cls, provider, title, creator, description):
 
     @classmethod
     def create_version(cls, create_from_guid, auth):
-        source_preprint = cls.load(create_from_guid.split(cls.GUID_VERSION_DELIMITER)[0])
+        base_guid = Guid.load(create_from_guid)
+        source_preprint = cls.load(base_guid._id)
         if not source_preprint:
             raise NotFound
         if not source_preprint.has_permission(auth.user, ADMIN):
             raise PermissionsError
-        if not source_preprint.is_published:
+        if not source_preprint.date_published:
             raise PendingPreprintVersionExists
```

Comment:
Hmm, after we made the guid always points to the latest version,`source_preprint` (when it is not the initial version) is not pending. I think we need a helper method to check if a preprint has a pending version, which should be as easy as check the if the version with the largest number is pending or not.